internal/store/postgres/notification_test.go

```diff
@@ -118,6 +128,145 @@ func (s *NotificationRepositoryTestSuite) TestCreate() {
 	}
 }
 
+func (s *NotificationRepositoryTestSuite) TestList() {
+	type testCase struct {
+		Description           string
+		Filter                notification.Filter
+		ExpectedNotifications []notification.Notification
+		ErrString             string
+	}
+
+	var testCases = []testCase{
+		{
+			Description: "should get all notification with receiver_selector filter",
+			Filter: notification.Filter{
+				ReceiverSelector: map[string]string{
+					"team": "gotocompany-infra",
+				},
+			},
+			ExpectedNotifications: []notification.Notification{
+				{
+					ID:          "789",
+					NamespaceID: 1,
+					Type:        "subscriber",
+					Data: map[string]interface{}{
+						"data-key": "data-value",
+					},
+					Labels: map[string]string{
+						"label-key": "label-value",
+					},
+					Template: "",
+					ReceiverSelectors: []map[string]string{
+						{
+							"team":     "gotocompany-infra",
+							"severity": "WARNING",
+						},
+						{
+							"id": "2",
+						},
+					},
+				},
+			},
+		},
+		{
+			Description: "should get all notification with type filter",
+			Filter: notification.Filter{
+				Type: "receiver",
+			},
+			ExpectedNotifications: []notification.Notification{
+				{
+					ID:          "123",
+					NamespaceID: 1,
+					Type:        "receiver",
```

Comment:
Let's update this to either `alert` or `event`.
Let's update the mock files too